relion/constants.py

```diff
@@ -178,3 +178,575 @@
     'rlnGroupNumber',
     'rlnRandomSubset'
 ]
+
+# Labels from relion-4.0/src/metadata_label.h as of 20-Feb-2022
+
+LABELS_DICT = {
+    #"rlnAccumMotionEarly": float,
```

Comment:
Can we remove the commented lines?

Reply:
I though it would be easier for the future to add more labels without digging into relion again

Reply:
Ok, that makes sense...let's keep it as it is then.